src/main/kotlin/calculator/Calculator.kt

```diff
@@ -0,0 +1,22 @@
+package calculator
+
+class Calculator(
+    private val parser: Parser
+) {
+    private var result: Int = 0
+
+    fun calculate(input: Input): Int {
+        val elements = parser.parse(input)
+        result = elements[0].toInt()
```

Comment:
elements[0] 대신에 `first()` 함수를 대신 써볼 수 있을 것 같아요. 😃 
```suggestion
        result = elements.first().toInt()
```

이 외에도 List의 확장함수로 자바보다 훨씬 더 유용한 함수들이 많으니 아래와 같이 어떤 함수들이있는지 쭉 훑어보고 잘 모르겠거나 궁금한 기능들은 수업 시간에 소개한 `학습테스트`를 작성하면서 공부해보시면 좋을 것 같아요. 💪 

<img width="844" alt="image" src="https://user-images.githubusercontent.com/47850258/200182778-3fca0149-471b-4782-ae36-063d8ff4e20d.png">

Reply:
감사해요~ 해당 부분은 학습테스트를 작성하면서 공부해볼게요